src/configs/index.js

```diff
@@ -0,0 +1,54 @@
+import ImportContactsIcon from '@material-ui/icons/ImportContacts';
+
+const languages = ['uk', 'en'];
+
+const routes = {
+  pathToNews: '/',
+  pathToNewsDetails: '/news/:id',
+  pathToAddNews: '/newsadd'
+};
+
+const theme = {
```

Comment:
Please, remove these variables and move its values to the object `config` where it is used

Reply:
I can't move routes to config object, because I use routes in menuCategories of config object.